### DISCUSSION TITLE
Intellisense not working on custom DLLs?

### DISCUSSION THREAD

**Discussion by bernatagulloesbrina** [TabularEditor/TabularEditor3#1105] [Ideas]
Hi, 

Not sure if it's supposed to work or not, but I could not get the intellisense of TE3 to work on methods from my custom dll. Is it something that will eventually come? 

Regards,

**Comment by otykier** [comment]
Thanks for reporting. It's a known limitation currently, but we are planning to make it work. However, please be aware that for comments to show up in the tooltips (such as in the screenshot below), the xml file containing the xmldoc comments must be present in the same folder as the dll file you're importing:

![image](https://user-images.githubusercontent.com/8976200/178343988-4399026a-5e51-4389-ad68-9fb354ef5bc2.png)
